cw/tvla.py

```diff
@@ -648,6 +682,10 @@ def run_tvla(ctx: typer.Context):
                 for i in range(num_traces):
                     leakage[i] = np.array_equal(fixed_key, keys[i])
 
+            # Uncomment the function call below for debugging e.g. when the t-test results aren't
+            # centered around 0.
+            # plot_fixed_vs_random(traces, leakage)
```

Comment:
Eventually we will probably want to add the plot_statistics argument and to modify tvla_cfg.yaml files.
Perhaps add a `#TO DO:` comment here so we don't forget

Reply:
I think we really just need that for debugging purposes and having yet another argument for this is probably not worth it.

Also, if we really wanted that, we should also switch the function implementation to compute the means and variances from the histogram data. Right now it may consume quite a lot of memory and compute. It's really just for debugging. Thus I would say let's not worry about that.